user-guide.rst

```diff
@@ -4,24 +4,28 @@ User Guide & FAQ
 Frequently Asked Questions
 --------------------------
 
-.. rubric:: Why are there different Scenario Explorer for specific projects?
+.. rubric:: Why are there different *Scenario Explorer* instances for specific projects?
 
 Each project usually brings together different consortium partners and uses a
```

Comment:
I'd include the points that each project covers different research questions and therefore produces different modeling results.